opacus/privacy_engine/factory.py

```diff
@@ -0,0 +1,33 @@
+from opacus import PrivacyEngine
+from opacus.privacy_engine.catalog import (
+    PrivacyEngine,
+    PrivacyEngineNonPoisson,
+    PrivacyEnginePerLayer,
+    PrivacyEnginePerLayerNonPoisson,
+)
+
+
+class PrivacyEngineFactory:
+    @classmethod
+    def get(
+        cls,
+        accountant: str = "rdp",  # TODO: str or enum?
+        clipping: str = "flat",  # TODO: str or enum?
+        poisson_sampling: bool = True,
+        secure_mode: bool = False,
+    ):
+        if accountant != "rdp":
```

Comment:
```suggestion
        if accountant.lower() != "rdp":
```

Reply:
I would recommend not to do so. The potential value is that it is easier to make the code runnable. 
The downside is: This promotes inconsistency in the code. There is a change we will have a pain of handling these differences in different parts of the codebase in the future. It will be nearly impossible to roll back after people start using different spellings.

Also, try to find an error:
solver="Ibfgs"
(hint: correct is solver="lbfgs")

Reply:
Touché. I withdraw my comment.